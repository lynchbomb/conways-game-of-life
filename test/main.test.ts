import { describe, expect, test } from 'vitest';

describe('needs a test describe', () => {
  test('needs a test', () => {
    expect(true).toBeTruthy();
  });
});
