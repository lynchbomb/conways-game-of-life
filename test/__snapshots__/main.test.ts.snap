// Vitest Snapshot v1

exports[`applyRules() > the new grid should have all cells alive 1`] = `
[
  [
    "X",
    "X",
    "X",
    "X",
  ],
  [
    "X",
    "X",
    "X",
    "X",
  ],
  [
    "X",
    "X",
    "X",
    "X",
  ],
  [
    "X",
    "X",
    "X",
    "X",
  ],
]
`;

exports[`generateRandomGrid() > grid cells should all be the same 1`] = `
[
  [
    "P",
    "P",
    "P",
    "P",
    "P",
  ],
  [
    "P",
    "P",
    "P",
    "P",
    "P",
  ],
  [
    "P",
    "P",
    "P",
    "P",
    "P",
  ],
  [
    "P",
    "P",
    "P",
    "P",
    "P",
  ],
  [
    "P",
    "P",
    "P",
    "P",
    "P",
  ],
  [
    "P",
    "P",
    "P",
    "P",
    "P",
  ],
]
`;
